README.md

```diff
@@ -320,6 +321,25 @@ config/initializers/passwordless.rb
 Passwordless.restrict_token_reuse = true
 ```
 
+### Supporting a database that uses UUIDs as its primary key type
+
+If your database uses a type other than `int` as the primary key type (e.g. you use `uuid`), you will need to add a migration
```

Comment:
I would mention _table_ rather than _database_ since you declare the type on table level.
```
create_table :users, id: :uuid do |t|
  ...
end
```

Reply:
I agree and we can probably shorten the title too to something like `Supporting UUID primary keys`.